bin/get_base_checksum.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser
+from docker_utils import get_labels
+
+parser = ArgumentParser()
+parser.add_argument(
+    "-n", "--name", dest="name", help="Docker image name e.g cmssw/el8:aarch64-runtime", type=str
+)
+args = parser.parse_args()
+
+labels = get_labels(args.name)
+print(labels['build-checksum'])
```

Comment:
@aandvalenzuela , this only works for our images and will not work for baseimage like `almalinux:8` .... right?

Reply:
Yes, only for the images in our dockerhub

Reply:
where are you going to use it? build container job should already know this checksum

Reply:
For example, when building `grid` images, the grid build container know its own checksum, but I need to get the checksum of the base `runtime`.